opentelemetry-api/src/opentelemetry/context/contextvars_context.py

```diff
@@ -14,14 +14,13 @@
 from contextvars import ContextVar
 from sys import version_info
 
-from opentelemetry.context import Context
-from opentelemetry.context.context import RuntimeContext
+from opentelemetry.context.context import Context, RuntimeContext
 
 if (3, 5, 3) <= version_info < (3, 7):
-    import aiocontextvars  # type: ignore # pylint:disable=unused-import
+    import aiocontextvars  # type: ignore # pylint:disable=unused-import,import-error
```

Comment:
Please note that without this patch in aiocontextvars https://github.com/fantix/aiocontextvars/pull/66 behavior of async code in python<3.7 differs from original contextvars in python 3.7.

Reply:
Hello @condorcet 

What specific behavior difference are you referring to?

Reply:
Hi @ocelotl !
Even with aiocontextvars `call_soon` and similar methods of loop don't create their own context. Hence such scheduled callbacks are "stealing" outer context.
For example:
```
import asyncio
from contextvars import ContextVar

var = ContextVar('var')

def callback(name):
    print(f'{name} var before: {var.get()}')
    var.set(name)
    print(f'{name} var after: {var.get()}')

if __name__ == '__main__':
    var.set('root')
    loop = asyncio.get_event_loop()
    loop.call_soon(callback, 'callback_1')
    loop.call_soon(callback, 'callback_2')
    asyncio.get_event_loop().run_forever()
```

In python 3.7 (proper behavior):
```
callback_1 var before: root
callback_1 var after: callback_1
callback_2 var before: root
callback_2 var after: callback_2
```

In python 3.6 (with aiocontextvars):
```
callback_1 var before: root
callback_1 var after: callback_1
callback_2 var before: callback_1
callback_2 var after: callback_2
```

Here you can see that first callback changed context var, and next callback used modified value rather than initial.

Reply:
Thanks for clarifying @condorcet, I'll open an issue to track this.

Reply:
Opened https://github.com/open-telemetry/opentelemetry-python/issues/436